### ISSUE TITLE
Using Projectile -> Levitation on another player extremely buggy

### ISSUE THREAD

**Issue by jzburda** [TeamWizardry/Wizardry#338]

**Comment by jzburda** [comment]
![2020-11-26_23 40 09](https://user-images.githubusercontent.com/6935020/100415536-fec0fc80-3041-11eb-9f38-f60c6fda6a19.png)
![2020-11-26_23 40 14](https://user-images.githubusercontent.com/6935020/100415537-fec0fc80-3041-11eb-914a-7484b0e99670.png)

**Comment by murapix** [comment]
Projectile -> Levitation is already not really going to work, Levitation practically _requires_ Beam to be the shape used to cast it. Also, what Wizardry version are you using, the bug about particles sticking around should have been fixed within the last couple patches.

**Comment by jzburda** [comment]
> 
> 
> Projectile -> Levitation is already not really going to work, Levitation practically _requires_ Beam to be the shape used to cast it. Also, what Wizardry version are you using, the bug about particles sticking around should have been fixed within the last couple patches.

Yeah me and my friend didn't know what we were doing with the whole shape thing. The version we have is 0.11.1

**Comment by murapix** [comment]
Hmm. Might have missed something with Levitation when doing the fixes, but oh well. It was a cool spell idea that really just didn't fit into the mod in the end, and it's not being added back in after our rewrite, so I honestly just wanna say "ignore Levitation" and leave it at that.

**Comment by jzburda** [comment]
> 
> 
> Hmm. Might have missed something with Levitation when doing the fixes, but oh well. It was a cool spell idea that really just didn't fit into the mod in the end, and it's not being added back in after our rewrite, so I honestly just wanna say "ignore Levitation" and leave it at that.

Is there a way I can remove your spell projectile entities from the world easily?
